Packs/CloudIDS/Playbooks/Cloud_IDS-IP_Blacklist-GCP_Firewall_README.md

```diff
@@ -0,0 +1,11 @@
+# Cloud_IDS-IP_Blacklist-GCP_Firewall
```

Comment:
I think that you create readme file for each playbook so this readme file could be removed.